src/ui/layout/constraints.md

```diff
@@ -1368,8 +1417,8 @@ But why does the `Container` decide that?
 Simply because that’s a design decision by those who
 created the `Container` widget. It could have been
 created differently, and you have to read the
-[`Container` documentation][] to understand how it
-behaves, depending on the circumstances.
+[`Container`][] API to understand how it behaves,
```

Comment:
you have to read the API documentation, not just the API. The API itself doesn't tell you anything.